cime/src/drivers/mct/main/seq_rest_mod.F90

```diff
@@ -480,6 +494,15 @@ subroutine seq_rest_write(EClock_d, seq_SyncClock, infodata, &
              call seq_io_write(rest_file, l2gacc_lx_cnt, 'l2gacc_lx_cnt', &
                   whead=whead, wdata=wdata)
           end if
+          if (glc_present) then
+             gsmap         => component_get_gsmap_cx(glc(1))
+             x2gacc_gx => prep_glc_get_x2gacc_gx()
```

Comment:
Same comment as above: This variable is only allocated in `prep_glc_mod` if `ocn_c2_glc` is true. Should this block of code be inside a `ocn_c2_glc` conditional?